state/redis/redis.go

```diff
@@ -298,8 +298,16 @@ func (r *StateStore) Multi(request *state.TransactionalStateRequest) error {
 	for _, o := range request.Operations {
 		if o.Operation == state.Upsert {
 			req := o.Request.(state.SetRequest)
-			b, _ := r.json.Marshal(req.Value)
-			pipe.Set(req.Key, b, defaultExpirationTime)
+
```

Comment:
changes similar to https://github.com/dapr/components-contrib/pull/460/files#diff-ae10e12c8f4b65b24b405cfe5a7ed386R255